crates/cdk/src/mint/mod.rs

```diff
@@ -859,28 +683,4 @@ mod tests {
             }
         }
     }
-
-    #[tokio::test]
-    async fn test_mint_keyset_gen() {
```

Comment:
Offline we talked about a test to ensure the signatory generates the same keyset as the current logic. This seems to be the test we want but its removed or is moved? I tested it manually with cdk-mintd and it is correct would be good to lock it in with a test.